ext/cl_khr_subgroups.asciidoc

```diff
@@ -250,17 +273,12 @@ given by `memory_order` apply to work items in a subgroup.
 This memory scope can be used when performing atomic operations to global or
 local memory.
 
-[[cl_khr_subgroups-additions-to-section-6.13.15-work-group-functions]]
-==== Additions to section 6.13.15 -- Work Group Functions
+[[cl_khr_subgroups-add-a-new-section-6.13.X-sub-group-functions]]
+==== Add a new section 6.13.X -- Sub-Group Functions
 
-The OpenCL C programming language implements the following built-in
-functions that operate on a subgroup level.
-These built-in functions must be encountered by all work items in a subgroup
-executing the kernel.
-We use the generic type name `gentype` to indicate the built-in data types
-`half` (only if the *cl_khr_fp16* extension is supported), `int`,
-`uint`, `long`, `ulong`, `float` or `double` (only if double
-precision is supported) as the type for the arguments.
+The table below describes OpenCL C programming language built-in functions that operate on a subgroup level.
+These built-in functions must be encountered by all work items in the subgroup executing the kernel.
+For the functions below, the generic type name `gentype` may be the one of the supported built-in scalar data types `int`, `uint`, `long`, `ulong`, `float`, `double` (if double precision is supported), or `half` (if half precision is supported).
```

Comment:
Just pointing out that the explicit mention of `cl_khr_fp16` is gone. I can see that becoming practical if we expose the functionality with a mechanism different from the extension (really thinking of https://gitlab.khronos.org/opencl/OpenCL-Docs/issues/106 here).

Reply:
Yup, exactly!